package.json

```
@@ -11,28 +11,28 @@
         "@openzeppelin/contracts": "4.9.0"
     },
     "devDependencies": {
-        "@1inch/solidity-utils": "2.2.27",
-        "@nomicfoundation/hardhat-chai-matchers": "1.0.6",
+        "@1inch/solidity-utils": "2.3.0",
+        "@nomicfoundation/hardhat-chai-matchers": "2",
```

```suggestion
        "@nomicfoundation/hardhat-chai-matchers": "2.0.1",
```

Also yarn.lock would need to be regenerated